digital_comms/ccam.py

```diff
@@ -289,15 +415,29 @@ def _small_cell_capacity(self):
 
     @property
     def capacity_margin(self):
+        """obj: The capacity margin in this postcode sector TODO
+        """
         capacity_margin = self.capacity - self.demand
         return capacity_margin
 
 
 def pairwise(iterable):
     """Return iterable of 2-tuples in a sliding window
 
+    Parameters
+    ----------
+    iterable: list
+        Sliding window
+
+    Returns
+    -------
+    list of tuple
+        Iterable of 2-tuples
+
+    Example
+    -------
         >>> list(pairwise([1,2,3,4]))
-        [(1,2),(2,3),(3,4)]
+            [(1,2),(2,3),(3,4)]
     """
     a, b = tee(iterable)
```

Comment:
What does this do?

Reply:
Its just an example that will be added to the documentation